sdks/java/core/src/main/java/org/apache/beam/sdk/util/RowJson.java

```diff
@@ -86,7 +88,7 @@
 })
```

Comment:
Please update the docstring for the newly supported type. It should probably also indicate what datetime formats are supported.

Reply:
Done.